README.md

```diff
@@ -459,6 +459,17 @@ end
 
 counter_culture now supports polymorphic associations of one level only.
 
+To discover which models need to be updated via `counter_culture_fix_counts`,
+counter_culture performs a `DISTINCT` query on the polymorphic relationship.
+This query can be expensive so we therefore offer the option
+(`polymorphic_classes`) to specify the models' counts that should be corrected:
+
+```ruby
+Image.counter_culture_fix_counts(polymorphic_classes: Product)
+# or
+Image.counter_culture_fix_counts(polymorphic_classes: [Product, Employee])
```

Comment:
Note: I worked with @kleinjm on this issue. It would be neat if this could be inferred from the Rails API, but we haven't found an easy way. Open to suggestions!